basis/command-line/command-line-docs.factor

```diff
@@ -112,7 +113,9 @@ $nl
 ARTICLE: ".factor-roots" "Additional vocabulary roots file"
 "The vocabulary roots file is named " { $snippet ".factor-roots" } ". If it exists, it is loaded every time Factor starts. It contains a newline-separated list of " { $link "vocabs.roots" } "."
 $nl
-"A word to run this file from an existing Factor session:"
+"A word to create this file:"
+{ $subsections scaffold-factor-roots }
```

Comment:
I think this would cause ``tools.scaffold`` to be loaded (since this creates a new dependency), not sure we want that...